### ISSUE TITLE
command not found: nmake

### ISSUE THREAD

**Issue by istvandesign** [spmeesseman/vscode-taskexplorer#89]
I am trying to use this with makefiles and on all scripts it returns `command not found: nmake. ` 
If running make run X it works.

**Comment by hangtwenty** [comment]
Yeah, the default setting _should_ (arguably) be `make`. Considering that `make` is the original command that `Makefile`s are for, and if this `nmake` command supports `Makefile`s that's fine but it's secondary

**Comment by hangtwenty** [comment]
Ah, so there is some code that tries to vary this by the platform, with a default to `make` (and only uses `nmake` if it detects the platform is Windows) --

https://github.com/spmeesseman/vscode-taskexplorer/blob/490faef96dd9b14e32a484e74dd430aeeac66053/src/taskProviderMake.ts#L203-L213

I'm on Mac and haven't taken in any settings from Windows computer(s) though so I'm not sure why I got that. Maybe it is something to do with the `package.json` but I'm not sure what would read that value (I am a newcomer to the VS Code ecosystem)

https://github.com/spmeesseman/vscode-taskexplorer/blob/fcc841882d0ade16ce35c2f86078ff0f8d99bb01/package.json#L889

**Comment by valoricDe** [comment]
Workaround
Go to Code -> Pref.. -> Settings -> Extensions -> Task Manager -> Edit Settings.json add
`{
  ...
  "taskExplorer.pathToMake": "make"
}`

**Comment by emil14** [comment]
Same issue with Ubuntu Linux 18.04

**Comment by vedro-compota** [comment]
Same issue with **Ubuntu 20.04**

**Comment by waltcow** [comment]
Same issue with Debian 11

**Comment by Hepson** [comment]
To update @valoricDe's workaround due to some changes https://github.com/spmeesseman/vscode-taskexplorer/commit/bd1c093743fe2307b6fccfb3e390ed5a46c887d1#diff-3dfac35103bf59e55ba06f427d7800177fbc10f156f8d55b0782c6a94cf2e847

We should now use `taskExplorer.pathToPrograms` as `{ "settings": { "taskExplorer.pathToPrograms": { "make": "make", }, } }`
